packages/launcher/package.json

```diff
@@ -4,9 +4,8 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "build": "tsc --project .",
```

Comment:
Why change these from `build` to `build-js`? The original idea was for each package to have a `build` script, so that running it in any given package would have it properly built for development. Running `yarn` in the root should then build every package in the `postinstall`, so running `yarn start` works afterwards. Does that still work?

Reply:
since the launcher is uses ts-node in development, you shouldn't run build locally, since then require will pick up the .js instead of the .ts since they are emitted alongside the source. this is one reason why we need to run `git clean -dxf` so often

Reply:
and for the reporter package, running `build` does not affect anything but the reporter cypress tests, since the runner requires the reporter's source instead of the dist

Reply:
done